### PR TITLE
Add polysync lidar translator example and README

### PR DESCRIPTION
**Pull request by LucasBuckland**
Prior to this commit the PolySync C++ examples did not include anything that showed users how to use PolySync to ROS bridge headers to compile a project with custom PolySync message types in ROS. This commit adds an example which compiles from the generated ROS bridge headers and re-publishes custom PolySync lidar messages as the native ROS PointCloud2 message type. This commit also adds a README which explains how to compile and use the example.

### CODE REVIEW

File: PolysyncLidarTranslator/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    $ source /opt/ros/indigo/setup.bash
+    $ catkin_create_pkg polysync_lidar_translator std_msgs roscpp tf
+
+Copy "polysync_lidar_translator_node.cpp" to the src directory of the polysync_lidar_translator package you created.
+
+    $ cp PolySync-Core-CPP-Examples/PolysyncLidarTranslator/polysync_lidar_translator_node.cpp ~/catkin_ws/src/polysync_lidar_translator/src/
+
+Make sure that in the "Build" section of the CMakeLists file for your PolysyncLidarTranslator project are uncommented. All other lines in the "Build" section should be commented.
+
+    ## Declare a C++ executable
```
Comment (ltbj) [inline PolysyncLidarTranslator/README.md]: These `#` are going to be parsed by markdown as headers unless you have them in a code block.

Comment (LucasBuckland) [reply PolysyncLidarTranslator/README.md]: They actually don't strangely, the tab over causes it to be displayed properly as a code block if you look at it with the github gui. However, I will change this to make it more explicit.

Comment (LucasBuckland) [reply PolysyncLidarTranslator/README.md]: Done.

File: PolysyncLidarTranslator/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+To properly build this example with the generated PolySync messages in ROS, you will first need to set up your catkin workspace with the ros_bridge.
```
Comment (ltbj) [inline PolysyncLidarTranslator/README.md:13]: Please add a one or two sentence introduction to what the example does, and the value it provides. Listing other use cases on how to extend the example would add a lot of value.

Comment (LucasBuckland) [reply PolysyncLidarTranslator/README.md:13]: I have something similar to that at the bottom, I'll move it up.

File: PolysyncLidarTranslator/polysync_lidar_translator_node.cpp [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+    br.sendTransform( tf::StampedTransform(
+            transform,
+            ros::Time::now(),
+            "world",
+            frame_id ) );
+}
+
+
+// CallBack for PolySync Lidar Targets
```
Comment (ltbj) [inline PolysyncLidarTranslator/polysync_lidar_translator_node.cpp]: Lidar *points

File: PolysyncLidarTranslator/polysync_lidar_translator_node.cpp [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+    ros::Time timer;
+
+    // Get the source GUID as a string
+    std::ostringstream src_guid_ss;
+    src_guid_ss << msg->ps_header.src_guid;
+    std::string src_guid = src_guid_ss.str();
+
+    // get PS timestamp in nanoseconds
+    long long ps_nanoseconds = msg->ps_header.timestamp*1000;
```
Comment (ltbj) [inline PolysyncLidarTranslator/polysync_lidar_translator_node.cpp]: This is an `unsigned long long` in the PolySync data model.

**Comment by LucasBuckland** [comment]
Above style comments should be fixed in my last commit.

**Review by ltbj — APPROVED** [review]

**Pull request merged**